sshd-common/src/main/java/org/apache/sshd/common/config/ConfigFileReaderSupport.java

```diff
@@ -64,7 +68,26 @@ public final class ConfigFileReaderSupport {
     public static final String DEFAULT_KBD_INTERACTIVE_AUTH = "yes";
     public static final boolean DEFAULT_KBD_INTERACTIVE_AUTH_VALUE = parseBooleanValue(DEFAULT_KBD_INTERACTIVE_AUTH);
 
+    public static final String HOSTBASED_AUTH_CONFIG_PROTP = "HostbasedAuthentication";
+    public static final String DEFAULT_HOSTBASED_AUTH = "no";
+    public static final boolean DEFAULT_HOSTBASED_AUTH_VALUE = parseBooleanValue(DEFAULT_HOSTBASED_AUTH);
+
     public static final String PREFERRED_AUTHS_CONFIG_PROP = "PreferredAuthentications";
+    // NOTE: entries are ordered in the same order as the default setting for the client
+    public static final List<PreferredAuthConfigEntry> PREFERRED_AUTHS_CONFIG_ENTRIES = Collections
```

Comment:
Perhaps I'm missing something (can only review this in the Github Web UI right now), but I don't see any use of this list.